dev-lang/gnucobol/gnucobol-2.2.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="an open-source COBOL compiler"
+HOMEPAGE="https://open-cobol.sourceforge.io"
+SRC_URI="mirror://sourceforge/open-cobol/${P}.tar.gz"
+
+LICENSE="GPL-3+ LGPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE="berkdb nls readline"
+
+RDEPEND="dev-libs/gmp:0
+	sys-libs/ncurses:0
+	berkdb? ( sys-libs/db:4.8 )"
+DEPEND="${RDEPEND}
+	sys-devel/libtool"
```

Comment:
Quoting from https://sourceforge.net/projects/open-cobol/files/: 
   ***
    NOTE - libltdl is NOT needed when installing on Linux,
    SUN Solaris, MAC OS, CentOS or Windows
        (including Cygwin, MingW and native windows).
    It is also NOT needed with later versions of AIX and HP-UX.
    (AIX >= 5.1 and HP-UX >= 11.1 are known to NOT require this).
    (Check if you have the "dlopen" function).
    ***

So is libtool needed or not?